.github/workflows/build-release.yml

```diff
@@ -0,0 +1,71 @@
+name: 'RELEASE: Build release'
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'New version number eg: 5.3.1'
+        required: true
+        type: string
+
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  build-release:
+    name: Build release
+    runs-on: Ubuntu-22.04
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4.2.2
+
+      - name: Setup Node
+        uses: actions/setup-node@v4.1.0
+        with:
+          node-version-file: .nvmrc
+
+      - name: Install dependencies
+        run: npm ci
+
+      - name: Validate version
+        uses: actions/github-script@v7.0.1
+        with:
+          script: |
+            const { validateVersion } = await import('${{ github.workspace }}/.github/workflows/scripts/changelog-release-helper.mjs')
+
+            await validateVersion('${{ inputs.version }}')
+
+      - name: Update package version
+        run: npm version --no-git-tag-version --workspace govuk-frontend ${{ inputs.version }}
+
+      - name: Update CHANGELOG
+        uses: actions/github-script@v7.0.1
+        with:
+          script: |
+            const { updateChangelog } = await import('${{ github.workspace }}/.github/workflows/scripts/changelog-release-helper.mjs')
+
+            await updateChangelog('${{ inputs.version }}')
+
+      - name: Generate release notes
+        uses: actions/github-script@v7.0.1
+        with:
+          script: |
+            const { generateReleaseNotes } = await import('${{ github.workspace }}/.github/workflows/scripts/changelog-release-helper.mjs')
+
+            await generateReleaseNotes()
+
+      - name: Build release
+        run: npm run build:release
+
+      - name: Create pull request
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          git config --global user.name 'govuk-design-system-ci'
+          git config --global user.email 'design-system-developers@digital.cabinet-office.gov.uk'
```

Comment:
Similarly here, I kinda like having this email address here, but we should consider the implication for spam/phishing to our email addy.

P'raps we could use:

```
git config --global user.name '${{github.actor}}'
git config --global user.email '${{github.actor}}@users.noreply.github.com'
```